template/package.json

```diff
@@ -16,11 +16,15 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.10",
+    "@react-navigation/bottom-tabs": "^6.5.4",
     "@react-navigation/native": "^6.0.13",
     "@react-navigation/native-stack": "^6.9.1",
+    "@react-navigation/stack": "^6.3.12",
     "@reduxjs/toolkit": "^1.8.6",
     "i18n-js": "^4.1.1",
```

Comment:
Please remove i18n-js because you changed it to i18next

Reply:
removed